docs/api/core/Raycaster.html

```diff
@@ -175,6 +175,9 @@ <h3>[method:Array intersectObject]( [page:Object3D object], [page:Boolean recurs
 		<p>
 		*Note* that for meshes, faces must be pointed towards the origin of the [page:.ray ray] in order to be detected; intersections of the ray passing through the back of a face will not be detected. To raycast against both faces of an object, you'll want to set the [page:Mesh.material material]'s [page:Material.side side] property to *THREE.DoubleSide*.
 		</p>
+		<p>
+		*Note* that the raycaster have a first collision filter with the *cached .boundingSphere* of a [page:Geometry Geometry] or [page:BufferGeometry BufferGeometry], which is not updated by the [page:Geometry Geometry.*NeedUpdate] or [page:BufferAttribute.needsUpdate BufferAttribute.needsUpdate] flags. Consider calling [page:Geometry Geometry.computeBoundingSphere()] or [page:BufferGeometry BufferGeometry.computeBoundingSphere()] when modifying vertices dynamically.
```

Comment:
There are some issues with your PR. Please ensure that the link are parsed correctly.

![image](https://user-images.githubusercontent.com/12612165/36483721-348a0f92-1717-11e8-94ca-0b0a257b248f.png)

You can verify your PR with: https://rawgit.com/chadiik/three.js/patch-1/docs/index.html#api/core/Raycaster